.github/workflows/simple.yaml

```
@@ -101,6 +101,19 @@ jobs:
           R CMD check ${EXTRA_CHECK_FLAGS} ${{ steps.package-name.outputs.pkgbuild }}
         shell: bash
 
+      - name: Upload logs artifact 🗞️
+        uses: actions/upload-artifact@v4
+        if: always()
```

As a part of the exercises I want the attendees to enable this

```suggestion
        if: false
```